src/index.html

```diff
@@ -1,13 +1,436 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
+    <title>Brand of eco-cosmetics</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="./images/icons/logo.svg"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header>
+      <section class="header container">
+        <nav class="header__nav">
+          <a href="#menu" class="header__menu">
+            <img
+              src="images/icons/Menu.svg"
+              alt="menu burger"
+              class="header__menu-burger"
+            >
+          </a>
+
+          <a href="#" class="header__logotipe">
+            <img
+              src="images/icons/logotipe.svg"
+              alt="witch of mountain winds"
+              class="header__image"
+            >
+          </a>
+
+          <a href="#" class="header__logo">
+            <img
+              src="images/icons/logo.svg"
+              alt="witch of mountain winds"
+              class="header__mobile-logo"
+            >
+          </a>
+
+          <div class="header__bar">
+            <ul class="header__list">
+              <li class="header__item">
+                <a href="#shop" class="header__link">Shop</a>
+              </li>
+              <li class="header__item">
+                <a href="#contact" class="header__link">Contact</a>
+              </li>
+              <li class="header__item">
+                <a href="#about" class="header__link">About</a>
+              </li>
+            </ul>
+
+            <a href="#" class="header__bag page__icon"></a>
+          </div>
+        </nav>
+      </section>
+    </header>
+    <main>
+      <section class="face">
+        <div class="face__picture"></div>
+
+        <div class="face__description container">
+          <h1 class="face__title">Brand of eco-cosmetics and magic candles</h1>
+
+          <p class="face__info">20% of sales at a wild bird shelter</p>
+
+          <button onclick="window.location.href='#product'" class="face__button page__button">shop</button>
```

Comment:
Avoid using inline JavaScript. It's better to move the JavaScript code to a separated .js file.